packages/host/app/lib/sqlite-adapter.ts

```diff
@@ -215,6 +215,12 @@ export default class SQLiteAdapter implements DBAdapter {
       .replace(/ANY_VALUE\(([^)]*)\)/g, '$1')
       .replace(/CROSS JOIN LATERAL/g, 'CROSS JOIN')
       .replace(/ILIKE/g, 'LIKE') // sqlite LIKE is case insensitive
+      .replace(
```

Comment:
will transform `jsonb_array_elements_text(deps) AS jsonb_array_unnested` to `json_each(deps)` (sqlite fails with alias so we remove it)